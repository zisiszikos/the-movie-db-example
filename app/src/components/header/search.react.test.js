import React from 'react';
import Search from './search.react.js';

describe('Search component', () => {

    it('should render properly', () => {

        const search = shallow(<Search />);
        expect(search.is('.search')).toBe(true);
        expect(search.state().query).toEqual('');
        expect(search).toMatchSnapshot();

        search.find('input').simulate('change', {
            target: {
                value: 'fight'
            }
        });
        expect(search.state().query).toEqual('fight');
    });
});
