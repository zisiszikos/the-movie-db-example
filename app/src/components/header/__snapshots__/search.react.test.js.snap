// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search component should render properly 1`] = `
<div
  className="search"
>
  <i
    className="material-icons"
  >
    search
  </i>
  <input
    onChange={[Function]}
    placeholder="Search for movies..."
    type="text"
  />
</div>
`;
